kong/tools/wrpc/future.lua

```diff
@@ -16,6 +18,7 @@ local _MT = { __index = _M, }
 
 local function finish_future_life(future)
   future.responses_tab[future.seq] = nil
+  yield()
```

Comment:
seems not necessary?

Reply:
I've changed this to do this after `future:done()`.